java/ant.debugger/src/org/netbeans/modules/ant/debugger/IOManager.java

```diff
@@ -22,6 +22,7 @@
 import java.io.IOException;
 import java.util.Hashtable;
 import java.util.LinkedList;
+import java.util.List;
```

Comment:
I'm curious, why add the import if it is not used within the changeset?

Reply:
I think that is an artifact of the work.. I'll fix it.. Thanks!